pkg/resources/object_parameter.go

```diff
@@ -91,6 +93,9 @@ func CreateObjectParameter(d *schema.ResourceData, meta interface{}) error {
 	db := meta.(*sql.DB)
 	key := d.Get("key").(string)
 	value := d.Get("value").(string)
+	client := sdk.NewClientFromDB(db)
+	ctx := context.Background()
+	parameter := sdk.ObjectParameter(key)
 
 	parameterDefault := snowflake.GetParameterDefaults(snowflake.ParameterTypeObject)[key]
```

Comment:
Old SDK here too.

Reply:
I don't think we can get rid of that in this case, at least not fully. In order to make use of new sdk here, we need to have new sdk implementation for all objects which can take parameters and that's not the case just yet. It can be improved in the future, but I think it has to use old sdk for now